### PR TITLE
Add cache setting dropdown

### PR DESCRIPTION
**Pull request by MrVauxs**

### CODE REVIEW

**Comment by a-viv-a** [comment]
This does not _seem_ to work:
image
image

**Comment by MrVauxs** [comment]
Fixed it

File: js/navigation.js [hunk 1]
```diff
@@ -162,37 +162,38 @@ class NavBar {
 				title: "Add the site to your home screen. When used in conjunction with the Preload Offline Data option, this can create a functional offline copy of the site.",
 			},
 		);
-		this._addElement_label(NavBar._CAT_SETTINGS,
+		this._addElement_dropdown(NavBar._CAT_SETTINGS, NavBar._CAT_CACHE, {isSide: true});
```
Comment (a-viv-a) [inline js/navigation.js:165]: Can we make this ``{isSide: false}`` so that there is not overflow on many devices? Otherwise LGTM

**Comment by MrVauxs** [comment]
For @TheGiddyLimit to fix

pon., 16 maj 2022, 01:27 użytkownik Isaac ***@***.***>
napisał:

**Comment by TheGiddyLimit** [comment]
Noted. Working As Intended:tm: for now, though, so LGTM2 🙂

**Pull request merged**